INSTALL.md

```diff
@@ -143,7 +143,7 @@ Install on Manjaro
 
 For Manjaro 18 and later:
 
-    sudo pacman -Sy hw-probe
```

Comment:
How to refresh the database before searching for hw-probe? Otherwise pacman just won’t find the hw-probe package.

Is it OK to add `pacman -Sy` command before installing the package?

Reply:
No, `-y`(sync) should only be used with `-u (update)`; i.e., `-Syu`:

```
sudo pacman -Syu hw-probe
```

The next update of the Manjaro package will include `edid-decode` as a dependency.

Actually you were right about the instructions for the AUR package using `makepkg`. An alternative method would be using an AUR helper like `yay`:

```
yay -S hw-probe
```